tools/deploy.py

```diff
@@ -0,0 +1,447 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
What is the difference between deploy and test_deploy, I think it should be explained in docstr